refs/Product-design-principles.md

```diff
@@ -29,3 +29,14 @@ THIRD: if and only if no option exists of FIRST and SECOND type, can options hav
 ## 2. Minimal Decision Principle
 
 The user should not have to make choices regarding options or functionalities that are of no use to him. Ideally, the existence of functionalities and options should come to the user’s attention when and only when the user needs them.
+
+## 3. Privacy UX Enforcement Principle
+
+Any developer oriented product should always help 3rd-party developers and companies to continuously improve the UX of their product.
+
+This implies that all final user facing parts of the devkit and associated products are built in such a way that they:
+
+1. **always** enforce Privacy UX patterns and [privacy-by-design principles](https://en.wikipedia.org/wiki/Privacy_by_design#Foundational_principles)
```

Comment:
Maybe add also, in addition to general privacy by design patters, that it should enforce our own ["privacy-enabled connectedness" principles](https://github.com/blindnet-io/product-management/blob/main/refs/notion-of-privacy/notion-of-privacy.md#privacy-enabled-connectedness)

Fee free to add or not. Otherwise I like this.

Reply:
Done :+1: